lib/utils/cookie.js

```diff
@@ -22,7 +23,7 @@ cookie.prototype.get = function(str){
   var data = {};
 
   if (Cookies.get(this.config.key)) {
-    data = json.parse( decodeURIComponent(Cookies.get(this.config.key)) );
+    data = json.parse(Cookies.get(this.config.key));
   }
   if (str) {
     return ('undefined' !== typeof data[str]) ? data[str] : null;
```

Comment:
this isn't exactly in the scope of the PR, but how would you feel about changing the order of the logic here (and some other places) while we're in the file? We usually do `typeof (something) ==! 'undefined'` in our other JS projects, and we use that order on line 56. I think it would help readability of this file a bit. (This is a nitpick)